database/seeds/DeliverySeeder.php

```diff
@@ -34,13 +35,19 @@ public function run()
                 'centre_id' => function () use (&$centre_ids) {
                     return array_pop($centre_ids);
                 },
+                // check the date
+                'dispatched_at' => function () {
+                    return Carbon::today()
+                        ->subMonths(rand(0, 11))
+                        ->subDays(rand(0, 30));
```

Comment:
If:
* The date is 1st February
* 11 months are subtracted
* 30 days are subtracted
...the result will be over a year in the past. Is this okay?